manager/assets/modext/widgets/security/modx.panel.groups.roles.js

```diff
@@ -69,6 +80,7 @@ Ext.extend(MODx.panel.GroupsRoles,MODx.FormPanel,{
                             ,useSplitTips: true
                             ,monitorResize: true
                             ,width: 270
+                            ,minWidth: 270
                             ,minSize: 270
                             ,maxSize: 400
```

Comment:
It makes sense to me to have the tree at the smallest usable size. The user then expands the view as needed when they drill down into the tree. If you only have one or two levels, it makes no sense to take half the available view width as the rest would not often be utilised.